packages/web-components/src/calendar/calendar.template.ts

```diff
@@ -154,90 +195,105 @@ export function secondaryPanelTemplate<T extends Calendar>(options: CalendarOpti
 export const template: ElementViewTemplate<Calendar> = html`
   <div class="control" @keydown=${(x, c) => x.handleControlKeydown(c.event as KeyboardEvent)}>
     <div class="date-view">
-      <div class="header">
-        ${calendarTitleTemplate()}
-        <div class="navicon-container">
-          <span
-            class="navicon-up"
-            part="navicon-up"
-            tabindex="1"
-            @click="${x => x.handleSwitchMonth(x.getMonthInfo().previous.month, x.getMonthInfo().previous.year)}"
-            @keydown="${(x, c) => x.handleNavIconKeydown(c.event as KeyboardEvent, 'primary', 'previous')}"
-          >
-            ${ArrowUp16}
-          </span>
-          <span
-            class="navicon-down"
-            part="navicon-down"
-            tabindex="2"
-            @click="${x => x.handleSwitchMonth(x.getMonthInfo().next.month, x.getMonthInfo().next.year)}"
-            @keydown=${(x, c) => x.handleNavIconKeydown(c.event as KeyboardEvent, 'primary', 'next')}
-          >
-            ${ArrowDown16}
-          </span>
-        </div>
-      </div>
-      ${calendarTemplate({
-        dataGrid: 'fast-data-grid',
-        dataGridRow: 'fast-data-grid-row',
-        dataGridCell: 'fast-data-grid-cell',
-      })}
-      <div class="footer" part="footer">
-        ${when(
-          x => !x.hasAttribute('monthPickerVisible'),
-          html` <div
-            class=${x => (x.isToday() ? 'slotted-link inactive' : 'slotted-link')}
-            tabindex="7"
-            @click="${(x, c) => x.handleGoToToday()}"
-            @keydown="${(x, c) => x.handleLinkKeydown(c.event as KeyboardEvent)}"
-          >
-            Go to today
-          </div>`,
-        )}
-      </div>
-    </div>
-    ${when(
-      x => x.hasAttribute('monthPickerVisible'),
-      html`<div class="secondary-panel">
+      <div class="calendar-container">
         <div class="header">
-          ${x => calendarSecondaryPanelTitleTemplate()}
+          ${calendarTitleTemplate()}
           <div class="navicon-container">
             <span
               class="navicon-up"
               part="navicon-up"
-              tabindex="5"
-              @click="${x =>
-                x.yearPickerOpen
-                  ? (x.yearPickerDecade = x.getYearPickerInfo().previousStart)
-                  : (x.monthPickerYear = x.getMonthPickerInfo().previous)}"
-              @keydown=${(x, c) => x.handleNavIconKeydown(c.event as KeyboardEvent, 'secondary', 'previous')}
+              role="navigation"
+              aria-label="Previous Month"
+              aria-describedby="Press enter to go to previous month"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby
The value should be an ID of container with text.
You could use aria-description instead --> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

Reply:
Oops, it looks like I forgot to remove that line. I was trying to get the helper texts to work, but I think we agreed that the labels themselves are descriptive.